aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -51,15 +53,18 @@ async fn main() -> Result<()> {
             release_config,
             test_dir,
             endpoint,
+            framework_git_rev,
```

Comment:
Does this override the git hash in the yaml file?

Reply:
No this is just for compiling the governance script. In rare cases (like in the last release), the old compiler cannot compile the latest aptos framework which the cli will use by default. (e.g: latest aptos framework has inline function but old compiler cannot support that.)

Reply:
Shouldn't it just use the compiler from the same branch as the framework version? It doesn't seem quite right to use a different version of the compiler.